### PR TITLE
Implement norm function for OpenVINO backend with support for various…

### PR DESCRIPTION
**Pull request by aaishwarymishra**
This pull request implements the `norm` function for the OpenVINO backend, replacing the previous placeholder that raised a `NotImplementedError`. The new implementation supports several common norms and handles different values for the `ord` argument.

Math operations support:

* Implemented the `norm` function in `keras/src/backend/openvino/math.py`, supporting L2, L1, max (infinity), and min (-infinity) norms, with handling for different `axis` and `keepdims` values.
* Added an import for `numpy` as `np` to support norm calculations.… orders

Fixes: openvinotoolkit/openvino#34127

### CODE REVIEW

File: keras/src/backend/openvino/math.py [hunk 1]
```diff
@@ -125,4 +126,28 @@ def solve(a, b):
...
+    elif ord == 1:
+        norm = ov_opset.reduce_l1(x, axis_tensor, keepdims).output(0)
+    elif ord == np.inf or ord == "inf" or ord == float("inf"):
+        norm = ov_opset.reduce_max(
+            ov_opset.abs(x).output(0), axis_tensor, keepdims
+        ).output(0)
+    elif ord == -np.inf or ord == "-inf" or ord == float("-inf"):
+        norm = ov_opset.reduce_min(
+            ov_opset.abs(x).output(0), axis_tensor, keepdims
+        ).output(0)
```
[the remark this answers is not in the document]
Comment (aaishwarymishra) [reply keras/src/backend/openvino/math.py]: Oh, I am not sure how to handle it, I searched numpy,tensorflow and torch backends all of them passed the ord directly as they natively support norm , but for openvino there are different methods for each so I tried to cover all possible values, I can remove them if needed.

**Comment by hertschuh** [comment]
Oh, and you'll need to approve the CLA too.

**Comment by aaishwarymishra** [comment]
> Thank you for implementing this!
> 
> Can you:
> 
> 1. Move this code to `keras/src/backend/openvino/linalg.py`, this is definition in this file is a mistake.
> 2. Delete this line to enable the testing: https://github.com/keras-team/keras/blob/master/keras/src/backend/openvino/excluded_tests.txt#L32

So I removed the line from excluded_tests.py and ran the tests and they passed, the tests are language agnostic right after removing that line it should work with openvino or I have to do something else?

**Comment by hertschuh** [comment]
> So I removed the line from excluded_tests.py and ran the tests and they passed, the tests are language agnostic right after removing that line it should work with openvino or I have to do something else?

So not all of linalg is implemented ord=nuc norm is not supported on OpenVino. Can add the following lines to
https://github.com/keras-team/keras/blob/master/keras/src/backend/openvino/excluded_concrete_tests.txt#L182 ?

```
LinalgOpsCorrectnessTest::test_norm_2_nuc
LinalgOpsCorrectnessTest::test_qr
LinalgOpsCorrectnessTest::test_solve
LinalgOpsCorrectnessTest::test_solve_triangular
LinalgOpsCorrectnessTest::test_svd
QrOpTest
```

**Comment by aaishwarymishra** [comment]
I have implemented it similar to tensorflow norm implementation

File: keras/src/backend/openvino/excluded_concrete_tests.txt [hunk 2]
```diff
@@ -178,6 +178,19 @@ TestMathErrors::test_stft_invalid_window
...
+LinalgOpsCorrectnessTest::test_solve
+LinalgOpsCorrectnessTest::test_solve_triangular
+LinalgOpsCorrectnessTest::test_svd
+LinalgOpsCorrectnessTest::test_det
+LinalgOpsCorrectnessTest::test_eig
+LinalgOpsCorrectnessTest::test_eigh
+LinalgOpsCorrectnessTest::test_inv
+LinalgOpsCorrectnessTest::test_lstsq
+LinalgOpsCorrectnessTest::test_lu_factor
+LinalgOpsCorrectnessTest::test_norm_2
```
Comment (hertschuh) [inline keras/src/backend/openvino/excluded_concrete_tests.txt]: Please remove this line: `LinalgOpsCorrectnessTest::test_norm_2`

The norm tests with 2 dimensions should pass, except of the `test_norm_2_nuc` ones (which are excluded above line 181).

Comment (aaishwarymishra) [reply keras/src/backend/openvino/excluded_concrete_tests.txt]: They were failing because the test_norm_2 was run with 2 and -2, which were for nuc so i just added them.
